### PR TITLE
Added step to place secret within workflow

### LINKED ISSUE #9360: content: Automation with LHCI
* Use this template to request updates to existing content.
* Delete any fields that aren't relevant to your issue. 
* Update the title of this issue with a brief summary of the work to be done.

### What page(s) need to be updated?

[https://web.dev/lighthouse-ci/](https://web.dev/lighthouse-ci/)

### Why is this update needed?

Instructions miss a step to add repository secret that is required for the Status Check to work correctly

### What's the deadline?

none

### PR DESCRIPTION
**Pull request by mvogelgesang**
Fixes #9360 

Changes proposed in this pull request:

- Updated instructions to add the repository secret into the github workflow file

Individual CLA signed, Mark Vogelgesang

### CODE REVIEW

**Comment by AaronForinton** [comment]
Hi @brendankenny, wondered if you could take a look at this content from the Lighthouse side?

File: src/site/content/en/blog/lighthouse-ci/index.md [hunk 1]
```diff
@@ -473,7 +473,18 @@ The steps below explain how to set up a status check for Lighthouse CI.
     field to the token that you copied in the last step and then click the **Add
     secret** button.
 
-7.  The status check is ready for use. To test it, [create a new pull
+7.  Go to the `lighthouse-ci.yaml` file and add the new environment secret just below the lhci run command
```
Comment (brendankenny) [inline src/site/content/en/blog/lighthouse-ci/index.md:476]:
```suggestion
7.  Go to the `lighthouse-ci.yaml` file and add the new environment secret to the "run Lighthouse CI" command.
```

**Review by brendankenny — APPROVED** [review]
small suggestion to make it clear in the text that it's adding to the existing command, not adding a new command, though the diff also communicates that. Otherwise LGTM!

File: src/site/content/en/blog/lighthouse-ci/index.md (hunk 1 above)
Comment (AaronForinton) [reply src/site/content/en/blog/lighthouse-ci/index.md:476]: Thank you!

**Review by AaronForinton — APPROVED** [review]

**Pull request merged**